src/store/creator/types.ts

```diff
@@ -33,4 +33,5 @@ export type CreatorState = {
     referenceId: string | null;
     staged: CreatorItem;
     status: CreatorStatus;
+    stroke: Stroke;
```

Comment:
`Stroke` seems drawing related, should this move to the drawing store? Another option would be to store the color here alone so that it could potentially be reused for regions and highlights. Do we care about the stroke size?

Reply:
Good point. I don't think we should share color between modes, actually. I'll move `stroke` to drawing store.